build-apply/action.yml

```diff
@@ -114,4 +114,8 @@ runs:
       run: |
         echo "[>>>] transform terraform outputs to environment variables"
         terraform output -json | jq 'to_entries | map("TF_\(.key|ascii_upcase)=\(.value.value|tostring)") | .[]' -r >> $GITHUB_ENV
-        echo "name_prefix=$(terraform output -raw name_prefix)" >> $GITHUB_OUTPUT
+        if terraform output name_prefix; then
+          echo "name_prefix=$(terraform output -raw name_prefix)" >> $GITHUB_OUTPUT
+        else
+          echo "name_prefix output not set. If you were expecting a name_prefix to be set, please fix this issue."
```

Comment:
Grace, we need this else condition. THe change you're doing doesn't change anything in your error.